src/melee/it/it_2725.c

```diff
@@ -8943,7 +8943,7 @@ void it_802798D4(Item_GObj* item_gobj, CommandInfo* cmd)
 void it_8027990C(Item_GObj* item_gobj, CommandInfo* cmd)
 {
     PAD_STACK(4);
-    it_80273648(item_gobj, cmd->u->unk33.unk1, cmd->u->unk33.unk0);
+    it_80273648(item_gobj, cmd->u->unk33.unk0, cmd->u->unk33.unk1);
```

Comment:
not sure why this got switched, gonna poke it a bit more to see if it was swapped for a good reason